user/billing-overview.md

```diff
@@ -20,13 +20,13 @@ The variety of plans provides you with the flexibility to choose the plan that s
 |Annual      | Only grandfathered | Very high concurrency limit<br />Paid macOS, Linux, Windows, and FreeBSD build minutes (credits)<br />Paid user licenses (only per users triggering the builds)<br /><br />Purchase in Travis CI or contact support [for plans over $3,300](/user/account/plan) in Travis CI |
```

Comment:
In line 19, the word Purchase should be in [] brackes so it will be shown as link.